src/main/java/appeng/util/InventoryAdaptor.java

```diff
@@ -73,6 +73,7 @@ public static InventoryAdaptor getAdaptor(Object te, final ForgeDirection d) {
         } else if (te instanceof TileEntityChest) {
             return new AdaptorIInventory(Platform.GetChestInv(te));
         } else if (te instanceof ISidedInventory si) {
+            final int[] slots = si.getAccessibleSlotsFromSide(d.ordinal());
```

Comment:
Does this need to be moved up like this? I don't see it being used any earlier than before the other changes

Reply:
I'm 99% sure that I did this during an earlier attempt to fix the issue. I must have left that change in accidentally